blocks/hooks/custom-styles.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, getWrapperDisplayName } from '@wordpress/element';
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockRandomAnchor, hasBlockSupport, getBlockType } from '../api';
+
+const CUSTOM_STYLE_ATTRIBUTES = [ 'backgroundColor', 'textColor' ];
+
+const hasCustomStyles = ( attributes ) => (
+	CUSTOM_STYLE_ATTRIBUTES.some( attr => attributes[ attr ] )
+);
```

Comment:
Hi @mcsf, I added explicit support for custom styles, but this verification still happens, because even if blocks have support for custom styles, this function still needs to execute to check if custom styles are in fact used. I changed the name to hasCustomStyleAttributes.